runtime/lua/vscode/util.lua

```diff
@@ -79,4 +79,22 @@ function M.debounce(func, time)
   end
 end
 
+-- Since Nvim 0.10.0, `virtcol2col` changed from returning the last byte of a
+-- multi-byte character to returning the first byte. However, we need the column
+-- of the last byte, which is consistent with the selected region in Nvim.
+-- See https://github.com/neovim/neovim/issues/29786
+if fn.has("nvim-0.10.0") == 0 then
```

Comment:
Will this break with nvim bumps the batch version?

Reply:
I didn't get what you meant.

Reply:
This is checking whether or not the user is on nvim 0.10.0, right? What happens when a user upgrades to 0.10.1?

Reply:
It's OK, `:h has`. It's called "has", not "is" 😃

Reply:
Yeah but wouldn't you want the substring to be "0.10."? (No trailing zero?) It might be better to parse this and see if it's 0.10 or greater, so we don't have to bump this when 0.11 releases

Reply:
has('nvim-0.10') and has('nvim-0.10.0') are the same; both indicate version 0.10.0 or higher. I prefer to use the full version number.

Reply:
Ah, I misunderstood `has` and assumed it was a substring match!